### PR TITLE
updated challenges with solutions

### PR DESCRIPTION
**Pull request by SunkissedQueen**

### CODE REVIEW

File: javascript/jest.md [hunk 1]
```diff
@@ -357,46 +357,293 @@ Use `fit()` to focus on an it-block without having to comment all other it-block
...
+  // declare function that takes in a number
+  const oddNumber = (number) => {
+    // conditional statement using modulo and strict equality operators to determine if odd return odd
+    if(number %2 !==0) { 
+      return "odd"
+    // if determine to be even return not odd
+    } else if(number %2 ===0) {
+      return "not odd"
+    }
+  }
```
Comment (sjproctor) [inline javascript/jest.md:473]: Couple spacing issues here on lines 467 and 470.

File: javascript/jest.md [hunk 2]
```diff
@@ -357,46 +357,293 @@ Use `fit()` to focus on an it-block without having to comment all other it-block
...
-  - Write the function that will make the test pass.
+//   Write the function that will make the test pass.
+  // declare function that takes in a string
+  const fruitColor = (string) => {
+    // conditional statement and return of banana using strict equality
+    if(string === "banana") {
+      return "yellow"
+    }     
+    // conditional statement and return of apple using strict equality
+    if(string === "apple") {
```
Comment (sjproctor) [inline javascript/jest.md:495]: This can be one conditional statement with only one `if` and the rest `else if`

File: javascript/jest.md [hunk 3]
```diff
@@ -357,46 +357,293 @@ Use `fit()` to focus on an it-block without having to comment all other it-block
...
+// declare function called oddOrEven that takes in a number
+const oddOrEven = (number) => {
+  // conditional statement using modulo and strict inequality operators to determine if odd return odd
+  if(number %2 !==0) { 
+    return "odd"
+  // if determine to be even return even
+  } else if(number %2 ===0) {
+    return "even"
+  }
+}
```
Comment (sjproctor) [inline javascript/jest.md:559]: Couple spaces here too: `(number % 2 !== 0)`

File: javascript/jest.md [hunk 4]
```diff
@@ -357,46 +357,293 @@ Use `fit()` to focus on an it-block without having to comment all other it-block
...
+    return num1 * num2
+  }
 
-11. Write the test for a function called `multiply` that takes two numbers as arguments and logs the result of one of the numbers multiplied by the other.
-  - Write the function that will make the test pass.
+// 12.  Write the test for a function called divisibleBy that takes two numbers as arguments and returns whether the first number is evenly divisible by the second so that divisibleBy(10, 5) logs "10 is evenly divisible by 5".
+  // a describe method that lists the name of the function
+  describe("divisibleBy", () => {
+    // a test/it method that describes what the function does
+    it("akes two numbers as arguments and returns whether the first number is evenly divisible by the second", () => {
```
Comment (sjproctor) [inline javascript/jest.md:599]: small typo in your `it` statement

**Review by sjproctor — CHANGES_REQUESTED** [review]
Woohoo! Looks really good. Just a couple really small fixes.

File: javascript/jest.md (hunk 4 above)
Comment (SunkissedQueen) [reply javascript/jest.md:599]: "t" has been located.

File: javascript/jest.md (hunk 3 above)
Comment (SunkissedQueen) [reply javascript/jest.md:559]: Spaces have been added

File: javascript/jest.md (hunk 2 above)
Comment (SunkissedQueen) [reply javascript/jest.md:495]: Wow. I definitely meant to put else if...copy and paste and forgot. I corrected the issue.

File: javascript/jest.md (hunk 1 above)
Comment (SunkissedQueen) [reply javascript/jest.md:473]: Modulo and I will act right one day. Corrected the spacing

**Review by sjproctor — APPROVED** [review]
Looks great!

**Pull request merged**